lib/linguist/languages.yml

```diff
@@ -772,12 +772,15 @@ C++:
   aliases:
   - cpp
   extensions:
-  - ".cpp"
```

Comment:
Looks like you missed the comment right at the top of this file:

https://github.com/github-linguist/linguist/blob/d190ad88af555bb2ee213aa64205f2b9a97a077d/lib/linguist/languages.yml#L17-L19

If `.c++` is now considered the primary extension for the language, then this change is fine, but if it's not (which I suspect is the case) `.cpp` needs to remain the first extension in the list.